pageserver/src/import_datadir.rs

```diff
@@ -403,3 +285,241 @@ fn import_wal<R: Repository>(
 
     Ok(())
 }
+
+pub fn import_basebackup_from_tar<R: Repository, Reader: Read>(
+    tline: &mut DatadirTimeline<R>,
+    reader: Reader,
+    base_lsn: Lsn,
+) -> Result<()> {
+    info!("importing base at {}", base_lsn);
+    let mut modification = tline.begin_modification(base_lsn);
+    modification.init_empty()?;
+
+    let mut pg_control: Option<ControlFileData> = None;
+
+    // Import base
+    for base_tar_entry in tar::Archive::new(reader).entries()? {
+        let entry = base_tar_entry.unwrap();
+        let header = entry.header();
+        let len = header.entry_size()? as usize;
+        let file_path = header.path().unwrap().into_owned();
+
+        match header.entry_type() {
+            tar::EntryType::Regular => {
+                // let mut buffer = Vec::new();
+                // entry.read_to_end(&mut buffer).unwrap();
+
+                if let Some(res) = import_file(&mut modification, file_path.as_ref(), entry, len)? {
+                    // We found the pg_control file.
+                    pg_control = Some(res);
+                }
+            }
+            tar::EntryType::Directory => {
+                info!("directory {:?}", file_path);
+                if file_path.starts_with("pg_wal") {
+                    info!("found pg_wal in base lol");
```

Comment:
Let's remove this message.
pg_wal directory can be present in basebackup. 
That's fine. We ignore wal files anyway.